### PR TITLE
add awsebcli

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge-admin please rerender

**Comment by hadim** [comment]
@conda-forge/help-python ready for review/merge

File: recipes/awsebcli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - botocore >=1.15
+    - cement ==2.10.12,<3
+    - colorama >=0.4
+    - future >=0.16.0
+    - pathspec >=0.5.9
+    - python-dateutil >=2.1
+    - requests >=2.20.1
+    - setuptools >=20.0
+    - semantic_version >=2.8.5
+    - six >=1.11.0
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:32]: From the metadata it should be
```suggestion
    - six >=1.11.0,>1.12.0
```

File: recipes/awsebcli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - termcolor >=1.1.0
+    - urllib3 >=1.24.1
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:34]:
```suggestion
    - urllib3 >=1.24.1,<1.25
```

File: recipes/awsebcli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - wcwidth >=0.1.7
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:35]:
```suggestion
    - wcwidth >=0.1.7,<0.2.0
```

File: recipes/awsebcli/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - PyYAML >=5.3.1
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:36]:
```suggestion
    - pyyaml >=5.3.1
```

File: recipes/awsebcli/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python
+    - botocore >=1.15
+    - cement ==2.10.12,<3
+    - colorama >=0.4
+    - future >=0.16.0
+    - pathspec >=0.5.9
+    - python-dateutil >=2.1
+    - requests >=2.20.1
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:29]:
```suggestion
    - requests >=2.20.1,<2.21
```

File: recipes/awsebcli/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    - setuptools >=20.0
+    - semantic_version >=2.8.5
+    - six >=1.11.0
+    - termcolor >=1.1.0
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:33]:
```suggestion
    - termcolor 1.1.0
```

File: recipes/awsebcli/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - botocore >=1.15
+    - cement ==2.10.12,<3
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:24]:
```suggestion
    - cement 2.8.2
```

File: recipes/awsebcli/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    - colorama >=0.4
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:25]:
```suggestion
    - colorama >=0.4,<1.0
```

File: recipes/awsebcli/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:22]:
```suggestion
    - python
    - docker-compose >=1.25.2, <1.26.0
```

File: recipes/awsebcli/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+    - botocore >=1.15
```
Comment (marcelotrevisani) [inline recipes/awsebcli/meta.yaml:23]: it seems this package also need ``blessed``
```suggestion
    - botocore >=1.15
    - blessed >=1.9.5
```

but there is no ``blessed`` on conda-forge, you will need to add it as well

if you want to, next time you can give a go to [grayskull :skull: ](https://github.com/marcelotrevisani/grayskull) to generate recipes for packages on pypi

Comment (hadim) [reply recipes/awsebcli/meta.yaml:23]: Thanks about grayskull, I didn't know!

**Comment by hadim** [comment]
@marcelotrevisani about the deps versions. I tried to use the exact same first but it ends up with a lot of conflicts so I preferred to remove the `=<` so this package can be used in envs with the newest package such as `docker-compose >1.26`.

**Comment by hadim** [comment]
Doing another attempt using grayskull: https://github.com/conda-forge/staged-recipes/pull/11694

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani about the deps versions. I tried to use the exact same first but it ends up with a lot of conflicts so I preferred to remove the `=<` so this package can be used in envs with the newest package such as `docker-compose >1.26`.

Sure, no problem then, we just like to follow as close as possible the directives from the upstream

**Comment by hadim** [comment]
Can I add `blessed` to this repo?

**Comment by hadim** [comment]
For blessed to build on Windows, those two packages need to be released first: https://github.com/conda-forge/staged-recipes/pull/11696

**Comment by hadim** [comment]
LGTM @marcelotrevisani

**Pull request merged**